### DISCUSSION TITLE
Open ports on HASSOS as re-streams from go2rtc as package are not accessibel as add-on from HomeAssistant.

### DISCUSSION THREAD

**Discussion by Ronnnnny** [blakeblackshear/frigate#6148] [Ask A Question] [answered]
I have installed the latest version of Frigate as an add-on in a separate instance of Homeassistant on a separate PI4 and it is all up and running.

Now I want to put the cameras "behind" go2rtc for re-streaming to my main home assistant instance or to VLC or web browsers and to frigate.  I can not access the streams from outside PI4 as if the ports are not open on HASSOS (my guess). In the frigate docs under "restream" there is this tiny sentence ....Port 8554 must be open.... but how do I do that?

**Comment by NickM-27** [comment, marked as answer]
you just enable the port in the addon configuration page in HA

**Reply by Ronnnnny** [reply]
Gosh you're fast...and an excellent answer! many thanks

**Comment by dbuezas** [comment]
I spent quite some time finding out why I couldn't see the cameras live anymore.
It may be a good idea to mention this in the docs, maybe [here](https://docs.frigate.video/guides/configuring_go2rtc)  :)

**Reply by dbuezas** [reply]
PR: https://github.com/blakeblackshear/frigate/pull/6209

**Comment by ianwood** [comment]
I just spent 2 hours trying to figure this out. The configuration page in the add-on didn't show the correct port options until I hit reset to defaults. Then the full list of ports came up, had to type them in and presto, restream and all the other goodness started working.
